core/execution/future/market_snapshot.go

```diff
@@ -66,6 +67,14 @@ func NewMarketFromSnapshot(
 	volumeDiscountService fee.VolumeDiscountService,
 ) (*Market, error) {
 	mkt := em.Market
+
+	if vgcontext.InProgressUpgradeFrom(ctx, "v0.73.2") {
```

Comment:
do we also need something similar when loading proposals from snapshot? as if there is a market update or market create active proposal it will become invalid after the upgrade.

Reply:
You're right, we certainly do.